idseq_pipeline/commands/accessionid2seq_functions.py

```diff
@@ -138,18 +149,19 @@ def generate_alignment_viz_json(nt_file, nt_loc_db, db_type, annotated_m8,
                 ad = groups.get(accession_id, {'reads': []})
                 sequence, ad['family_id'], ad['genus_id'], ad[
                     'species_id'] = seq_info
+
                 ref_start = int(metrics[-4])
                 ref_end = int(metrics[-3])
                 if ref_start > ref_end:  # SWAP
                     (ref_start, ref_end) = (ref_end, ref_start)
                 ref_start -= 1
-                prev_start = (ref_start - REF_DISPLAY_RANGE
-                              ) if (ref_start - REF_DISPLAY_RANGE) > 0 else 0
+
+                prev_start = 0
+                if (ref_start - REF_DISPLAY_RANGE) > 0:
+                    prev_start = (ref_start - REF_DISPLAY_RANGE)
```

Comment:
Since 0 is not the default or a common case, I think the code is not more readable by putting it first.   But I also think the version before your change wasn't as readable as it should have been.  In my view the best style here for Python is
```
    prev_start = ref_start - REF_DISPLAY_RANGE
    if prev_start < 0:
        prev_start = 0
```